web/docusaurus.config.js

```diff
@@ -108,10 +108,6 @@ module.exports = {
       //   it searches only in v1 docs if you are searching from v1 docs.
       //   We should enable it if we start doing versioning.
       // contextualSearch: true
-    },
-    gtag: {
-      trackingID: 'G-3ZEDH3BVGE',
-      anonymizeIP: true,
```

Comment:
This was already down below, and I had to remove it to compile. Ref: https://github.com/facebook/docusaurus/pull/5832

Reply:
hmm honestly I don't remember seeing this before, sounds like something for google analytics? But if we have it in some other place then probably it's ok